RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -27,11 +27,16 @@ import RevenueCat
 @MainActor class CustomerCenterViewModel: ObservableObject {
 
     typealias CustomerInfoFetcher = @Sendable () async throws -> CustomerInfo
+    typealias CurrentVersionFetcher = () -> String?
+
+    private lazy var currentAppVersion: String? = currentVersionFetcher()
 
     @Published
     private(set) var hasSubscriptions: Bool = false
     @Published
     private(set) var subscriptionsAreFromApple: Bool = false
+    @Published
+    private(set) var appIsLatestVersion: Bool = false
```

Comment:
Ah good catch! This is an oversight. I had intended for it to default to `true`. I have moved `defaultAppIsLatestVersion` out of the `didSet` block to the top of the file, because I'd like to define this in one place. I don't know if this is how it's normally done? (In Kotlin I would define it as a `const` in a `companion object`.) See https://github.com/RevenueCat/purchases-ios/pull/4169/commits/54c6ff6b55802eeca4f04269d56ef26433eef8f0

Reply:
I don't think declaring private variables in the global namespace is common practice. Would declaring it as a static variable within the class achieve what you're looking for? (The value will be associated with the type rather than an instance.)

```swift
class CustomerCenterViewModel {
  private static let defaultAppIsLatestVersion = true
```

This can then be accessed as `CustomerCenterViewModel.defaultAppIsLatestVersion` or `Self.defaultAppIsLatestVersion`.

Reply:
`private static let` is the thing I was looking for, thanks!